unified-runtime/source/adapters/level_zero/image.cpp

```diff
@@ -43,7 +45,17 @@ ur_result_t urBindlessImagesImageCopyExp(
   UR_ASSERT(!(pSrcImageDesc && UR_MEM_TYPE_IMAGE1D_ARRAY < pSrcImageDesc->type),
             UR_RESULT_ERROR_INVALID_IMAGE_FORMAT_DESCRIPTOR);
 
-  bool UseCopyEngine = hQueue->useCopyEngine(/*PreferCopyEngine*/ true);
+  // When we do a region copy from an image handle to USM with non-zero offest
+  // into a USM region, then copy engine would ignore the offset and always
+  // write data at the beginning of the USM allocation.
+  // On the other hand, when performing memory to memory copies if copy engine
```

Comment:
The way this work in v2 adapter is that the driver decides which engine to use. I'm assuming it's working fine on v2 without any changes?